luigi_pipeline/seqr_loading.py

```diff
@@ -109,6 +134,12 @@ def validate_mt(mt, genome_version, sample_type):
         :param sample_type: WGS or WES
         :return: True or Exception
         """
+
+        contig_check_result = contig_check(mt, 100)
```

Comment:
I do think that this should either be a keyword argument to the `validate_mt` function, or at the very lease a top-level constant. Hard coding numbers into blocks of code is bad practice because its really hard to find and update them later